### PR TITLE
Refactor load/save

### PR DESCRIPTION
**Pull request by JevinJ**
### Overview

Refactor of save/load system. Replaces _vtk_reader, _vtk_writer properties with static class constants. This way they can be referenced in testing while also being part of the class itself.

### CODE REVIEW

File: pyvista/core/pointset.py [hunk 1]
```diff
@@ -581,6 +563,9 @@ class UnstructuredGrid(vtkUnstructuredGrid, PointGrid, UnstructuredGridFilters):
 
     """
 
+    _READERS = {'.vtu': vtk.vtkXMLUnstructuredGridReader, '.vtk': vtk.vtkUnstructuredGridReader}
```
Comment (akaszynski) [inline pyvista/core/pointset.py:566]: Agreed.  That's a better spot for them.

File: pyvista/core/grid.py [hunk 2]
```diff
@@ -97,6 +87,9 @@ class RectilinearGrid(vtkRectilinearGrid, Grid):
 
     """
 
+    _READERS = {'.vtk': vtk.vtkRectilinearGridReader, '.vtr': vtk.vtkXMLRectilinearGridReader}
```
Comment (akaszynski) [inline pyvista/core/grid.py:90]: Class attributes should be lowercase.

Comment (JevinJ) [reply pyvista/core/grid.py:90]: Should they if they're constants though?

Comment (akaszynski) [reply pyvista/core/grid.py:90]: Valid point.  Thought that constants were only defined at the module level and didn't consider it at the class level.  This cleared it up for me:
https://stackoverflow.com/a/45702782/3369879

**Comment by akaszynski** [comment]
Even cleaner and you taught me about class constants.  Thanks!

**Pull request merged**